cf_remote/remote.py

```diff
@@ -514,6 +514,12 @@ def install_host(
         log.error("Installation failed - no package found!")
         return 1
 
+    if data.get("bin") and not data.get("bin").get("curl"):
+        log.error(
+            "Couldn't download remotely. Curl is not installed on host '%s'" % host
+        )
+        return 1
```

Comment:
`curl` is only needed for the `--remote-download` feature. Please put this inside the `if` below.